src/Mvc/Mvc.Core/src/Authorization/AuthorizeFilter.cs

```diff
@@ -121,7 +121,7 @@ private Task<AuthorizationPolicy> ComputePolicyAsync()
                         nameof(IAuthorizationPolicyProvider)));
             }
 
-            return AuthorizationPolicy.CombineAsync(PolicyProvider, AuthorizeData);
+            return AuthorizationPolicy.CombineAsync(PolicyProvider, AuthorizeData!)!;
```

Comment:
`CombineAsync` could return null, but I'm choosing to ignore that case for now for the sake of keeping this simple